src/AuthenticationPortal.Contracts/Errors/BaseException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace AuthenticationPortal.Contracts
+{
+    [Serializable]
+    public class BaseException : Exception
+    {
+        public int Code { get; set; }
+        public string Message { get; set; }
```

Comment:
add list of `info` as well

Reply:
But, in the contract we have defined only 2 responses, 1) code & 2) Message. Should I change the contract also and add additional list of info response? And also what will this list of info include?

Reply:
you will need it in validation right or not ?

Reply:
Right, Added It and updated Contracts
Done